Maker-Agents-Suit-Module/ESMAgent/src/agent.ts

```diff
@@ -1,28 +1,19 @@
-import provideESMJoinEventAgent from './join.event';
-import provideESMFireEventAgent from './fire.event';
-import { Finding, HandleTransaction, TransactionEvent } from 'forta-agent';
+import provideESMJoinEventAgent from "./join.event";
+import provideESMFireEventAgent from "./fire.event";
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
 
-const MakerDAO_ESM_CONTRACT = '0x29cfbd381043d00a98fd9904a431015fef07af2f';
-const JOIN_EVENT_ALERTID = 'MakerDAO-ESM-1';
-const FIRE_EVENT_ALERTID = 'MakerDAO-ESM-2';
+const MakerDAO_ESM_CONTRACT = "0x29cfbd381043d00a98fd9904a431015fef07af2f";
```

Comment:
according to the [latest release](https://chainlog.makerdao.com/api/mainnet/active.json), ESM address seems to be different ("MCD_ESM": "0x09e05ff6142f2f9de8b6b65855a1d56b6cfe4c58"). 
this one belongs to [the older version](https://changelog.makerdao.com/releases/mainnet/1.9.10/contracts.json)